data/json/items/tool/fire.json

```
@@ -161,7 +161,7 @@
     "type": "TOOL",
     "name": { "str": "matchbook" },
     "description": "A small book of matches with a coarse strike surface on the outside flap for lighting them.  Allows you to use various drugs like cigarettes, light explosives like Molotov cocktails, or light nearby items on fire.  Due to the matches being lightly glued to the matchbook, they can be removed, but not replaced on the book.",
-    "category": "containers",
+    "category": "container",
```

```suggestion
    "category": "container"
```
Resetting test suite